src/main/kotlin/org/kamiblue/client/command/commands/ModifiedCommand.kt

```diff
@@ -0,0 +1,32 @@
+package org.kamiblue.client.command.commands
+
+import net.minecraft.util.text.TextComponentString
+import net.minecraft.util.text.event.ClickEvent
+import net.minecraft.util.text.event.HoverEvent
+import org.kamiblue.client.command.ClientCommand
+import org.kamiblue.client.util.text.MessageSendHelper.sendChatMessage
+
+object ModifiedCommand: ClientCommand(
+    name = "modified",
+    description = "View modified settings in a module"
+) {
+
+    init{
+        module("module"){
+            execute("List changed settings"){
+
+                for (setting in it.value.settingList.filter { it.isModified() }) {
+                    val component = TextComponentString("${setting.name} has been changed to ${setting.value}")
```

Comment:
Just make it `continue` in the for loop if not modified instead of doing filtering before it which requires extra iteration,